lib/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = (options = {}) => tree => {
   options.tags = options.tags || ['fetch', 'remote']
   options.attribute = options.attribute || 'url'
   options.got = options.got || {}
+  options.stayTag = options.stayTag || false
```

Comment:
Can we please rename this option to `preserveTag` instead of `stayTag`?

Reply:
done